js/game.js

```diff
@@ -97,7 +97,7 @@ Game.displayChunk = function(key){
     for(var i = 0; i < map.layers.length; i++) {
         // You can load a layer from the map using the layer name from Tiled, or by using the layer
         // index
-        var layer = map.createStaticLayer(i, tiles, chunkX*32, chunkY*32);
+        var layer = map.createLayer(i, tiles, chunkX*32, chunkY*32);
```

Comment:
This doesn't need to be changed but the new API of Phaser is `createLayer` and it shows warnings in the console.